src/signature.rs

```diff
@@ -540,6 +573,7 @@ mod tests {
             }
 
             unsafe { sequoia_mechanism_free(m1) }
+            drop(fixture_home_locked);
```

Comment:
This shouldn't be necessary, as leaving from the scope will automatically does that.